src/types/integrations.js

```diff
@@ -2,6 +2,74 @@
 // See LICENSE.txt for license information.
 // @flow
 
+export type IncomingWebhook = {|
+    id: string,
+    create_at: number,
+    update_at: number,
+    delete_at: number,
+    user_id: string,
+    channel_id: string,
+    team_id: string,
+    display_name: string,
+    description: string,
+    username: string,
+    icon_url: string,
+    channel_locked: boolean,
+|};
+
+export type OutgoingWebhook = {|
+    id: string,
+    token: string,
+    create_at: number,
+    update_at: number,
+    delete_at: number,
+    creator_id: string,
+    channel_id: string,
+    team_id: string,
+    trigger_words: Array<string>,
+    trigger_when: number,
+    callback_urls: Array<string>,
+    display_name: string,
+    description: string,
+    content_type: string,
+    username: string,
+    icon_url: string,
+|};
+
+export type Command = {|
```

Comment:
`SystemCommand` would represent commands like `/shrug` and others ? Since it is not used in `src/actions/integrations.js` Do I need to create it at another place that the current file ? And last question, where can I find sample/guidance on the `SystemCommand` usage and definition  ? 

Sorry for the number of questions !

Reply:
No worry :)
You're right, like those built-in slash commands. 
I realized that it's also a `Command` type, except for the "method" field that also accepts an empty string.